traffic_control/clients/python/trafficops/tosession.py

```diff
@@ -1451,21 +1451,6 @@ def assign_delivery_services_to_federations(self, federation_id=None, data=None)
 		:raises: Union[LoginError, OperationError]
 		"""
 
-	@api_request(u'delete', u'federation_resolvers/{federation_id:d}/deliveryservices/'
```

Comment:
https://github.com/apache/trafficcontrol/blob/master/traffic_ops/app/lib/TrafficOpsRoutes.pm#L636

This was most likely a typo meant for that route ☝️ .

Reply:
Possibly, and I did consider changing it instead of removing it, but that appears to be removing a delivery service from a federation (which is in line with the method name) whereas the route in the method seems to be trying to remove it from a federation *resolver*. I don't really have a very good grasp on the difference, so I decided to just remove it rather than try to fix it.